setup.cfg

```
@@ -1,25 +1,109 @@
+[metadata]
+description = Metric logger for ML projects.
+name = dvclive
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+license = Apache-2.0
+license_files = LICENSE
+url = https://github.com/iterative/dvclive
+project_urls =
+    Documentation = https://dvc.org/doc/dvclive
+    Source = https://github.com/iterative/dvclive
+platforms=any
+authors = Paweł Redzyński
+author_email = pawel@iterative.ai
+maintainer = Iterative
+maintainer_email = support@dvc.org
```

Usually, the authors are copyright holder (and that's how the template considers it). If it's okay with @pared, let's go with how we do in `dvc-data` for example:

https://github.com/iterative/dvc-data/blob/303b33448694148ca0639ff1ec0d31fae44efb0d/setup.cfg#L10-L11

```suggestion
authors = Iterative
maintainer_email = support@dvc.org
```

(just trying to reduce future merge conflicts 🙂, please remember to update the `.cruft.json` too if we went with the above suggestion)

---

Especially that recently the most work has been done by @daavoo :D
If he is also ok with that we can modify it to this.